integration/singlearch/patch_test.go

```diff
@@ -111,7 +112,10 @@ func TestPatch(t *testing.T) {
 			// The scanning should look for the tag that Copa actually created
 			scanTag := tagPatched
 			if !reportFile && img.IsManifestList {
-				scanTag += "-amd64"
+				// Determine the host platform
+				hostPlatform := platforms.DefaultSpec()
+				platformArch := fmt.Sprintf("-%s", hostPlatform.Architecture)
+				scanTag += platformArch
```

Comment:
Instead of host platform which would always be `amd64` in our CI, I was thinking we could check for one of the present platforms in the image we are patching. For example, if we are patching `docker.io/arm64v8/redis:7.4.0-alpine`, there is no `amd64` platform in the manifest list, so we should check for the `arm64` platform tag that exists rather than `amd64`